view/frontend/web/js/productlabel-position.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ProductLabel
+ * @author    Houda EL RHOZLANE <houda.elrhozlane@smile.fr>
+ * @copyright 2019 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+
+define([
+        "jquery"
+    ], function($){
+
+        var widthOfParent = $("#labels").parent().width();
```

Comment:
As I said previously, this does not work very well on a product listing page where several products are displaying a label. On my environment only the first label is displayed. Probably because there are several `#labels` elements. You should use a class to work with.

Reply:
Done!